src/modules/Hyperlinking.js

```diff
@@ -0,0 +1,23 @@
+import { h, render } from 'preact';
+
+import App from '../components/App';
+
+export default class Hyperlinking {
+    constructor(quill) {
+        this.quill = quill;
+
+        this.quill.on('selection-change', this.onSelection.bind(this));
+    }
+
+    onSelection(range) {
+        if (range) {
```

Comment:
We should render null also when range is `null`. Currently when someone click outside editor popup is not disappearing.